mapper/.devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "image": "ballerina/ballerina-devcontainer:2201.10.0",
+  "customizations": {
```

Comment:
Do we need to add this [.devcontainer.json](https://github.com/ballerina-platform/module-ballerinax-financial.iso20022-to-swiftmt/pull/3/files#diff-c140fdb96d32e8a31526c5ae262ac369ad893354744905009d661c52c580a8ce) file?

Reply:
It was generated when project init, Lets remove it